src/styles/sidebar/components/annotation-license.scss

```diff
@@ -0,0 +1,23 @@
+@use "../../variables" as var;
+
+.annotation-license {
+  @include var.font-small;
+  border-top: 1px solid var.$grey-3;
+  padding: 0.5em 0;
+
+  &__link {
+    display: flex;
+    color: var.$grey-5;
+  }
+
+  &__icon {
+    width: 12px;
+    height: 12px;
+    margin-right: 1px;
+    color: var.$grey-5;
```

Comment:
Setting the color of the icon is not necessary in the unhovered state because it is inherited from the link. In the hovered state, this ends up overriding the color from the link, which I don't think is intentional?

Hover state (master):

<img width="324" alt="Screenshot 2019-12-04 07 28 48" src="https://user-images.githubusercontent.com/2458/70122214-21574a00-1668-11ea-8bf5-1214601f2950.png">

Hover state (this branch):

<img width="353" alt="Screenshot 2019-12-04 07 28 22" src="https://user-images.githubusercontent.com/2458/70122180-100e3d80-1668-11ea-8757-baf917ee9021.png">

Reply:
Thanks. This was a silly oversight of mine. Fixing!

Reply:
Hi Lyza - Did you make a change here but not push it?

Reply:
Precisely.